src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -49,6 +49,8 @@ def _get_default_install_location(exe_name):
 name_arg_type = CliArgumentType(options_list=('--name', '-n'), metavar='NAME')
 
 register_cli_argument('acs', 'name', arg_type=name_arg_type, help='ACS cluster name', completer=get_resource_name_completion_list('Microsoft.ContainerService/ContainerServices'))
+register_cli_argument('acs', 'resource_group', arg_type=CliArgumentType(options_list=('--resource-group', '-g'),), help='Name of the Azure resource group')
```

Comment:
Use `resource_group_name_type` here to keep this argument consistent throughout the CLI.
There are examples of it being used in this file.